src/shogun/lib/SGMatrix.h

```diff
@@ -306,9 +308,66 @@ template<class T> class SGMatrix : public SGReferencedData
 		/** Set matrix to a constant */
 		void set_const(T const_elem);
 
-		/** fill matrix with zeros */
+		/** Fill matrix with zeros */
 		void zero();
 
+		/**
+		 * Fill matrix with uniformly distributed randoms
+		 * Float Types: [0, 1)
+		 * Int Types: std::numeric_limit<T>::(min->max)
+		**/
+		// vectorized available
```

Comment:
i wonder whether doxygen correctly parses the above docstring if this comment is here?
could even just remove it

Reply:
done

Reply:
done